rfcs/20200619-dropout/README.md

```diff
@@ -0,0 +1,425 @@
+# Proposal for DNNL dropout primitive
```

Comment:
You may want to use oneDNN instead of DNNL.

Reply:
Fixed.